WordPress/Classes/Models/ReaderCrossPostMeta.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
 import CoreData
 
-@objc public class ReaderCrossPostMeta : NSManagedObject
+@objc public class ReaderCrossPostMeta : NSManagedObject, ManagedObject
```

Comment:
Yeah, I tried to do that but I think I hit a dead end(Xcode bug)

If I implement the protocol `ManagedObject` on `ReaderCrossPostMeta` in an extension I get an Xcode error(bug?)
![screen shot 2016-11-29 at 2 29 16 pm](https://cloud.githubusercontent.com/assets/562080/20725611/6c073cbc-b640-11e6-80ec-44c2818141fc.png)

And I can't implement the protocol on `ReaderPost` and `SourcePostAttribution` without generating warnings because they are `obj-c` classes and I can't import `WordPress-Swift.h` in a header file.

Would be be ok with lettings this be for the mean time @aerych ?